src/styles/index.scss

```diff
@@ -1,3 +1,137 @@
+@import './variables';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+ul li {
+  padding: 0;
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid $color-border;
+  background-color: $color-background;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  &__img {
+    margin: 0 2px 0 3px;
+    img {
+      height: 134px;
+      width: 160px;
+
+      margin-bottom: 36px;
+    }
+  }
+
+  &__title {
+    color: $color-text-brown;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+    margin-bottom: 4px;
+  }
+  &__product-code {
+    color: $color-text-gray;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+  &__stars-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-bottom: 24px;
+  }
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $color-text-brown;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    .price-title {
+      color: $color-text-gray;
+      font-size: 12px;
+      line-height: 18px;
+    }
+    .price {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $color-text-brown;
+    }
+  }
+  &__buy-button {
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-align: center;
+
+    color: $color-background;
+    background-color: $color-button;
+
+    &:hover {
+      background-color: $color-background;
+      border: 1px solid $color-button;
+      color: $color-button;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__item {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--1 .stars__item:nth-child(-n + 1) {
```

Comment:
All good but in this case you can use loop construction 
https://sass-lang.com/documentation/at-rules/control/for/
Which made you index for prefixes and nth-child.
@for $i from 1 through 5 {
 then use $i which will grow from 1 to 5.
}